bootstrap/py_bootstrap.bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+
+REM Check if Python is installed
+where python > nul 2>&1
+if %errorlevel% neq 0 (
+    echo Python is not installed
+    exit /b 1
+)
+
+REM Check if venv module is available
+python -m venv --help > nul 2>&1
```

Comment:
You could just remove the separate check step and handle the error when calling it normally below.

Reply:
good idea, https://github.com/apache/nifi-minifi-cpp/pull/1681/commits/194c8f3ab46266f43fbdf314e96759679b83b84b